pagopa/io-app
# chore: [IOBP-244] Addition of IDPAY CODE playground

## Short description
addition of said screen
<img width="250" alt="image" src="https://github.com/pagopa/io-app/assets/60693085/25093c5a-ba37-468c-a880-fd1e9ffd5bbe">

## List of changes proposed in this pull request
- added required navigators
- added two mock entries to be filled in the future

## How to test
with the TEST IDPAY flag active, navigate to the profile screen and make sure that the "IDPAY CODE PLAYGROUND" ListItem is rendered, then click it and make sure it is rendered as per provided image
